zuk.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<manifest>
-<project name="zuk-italian-devs/android_device_zuk_z2_plus" path="device/zuk/z2_plus" remote="github"  revision="8x-caf" />
-<project name="zuk-italian-devs/android_device_zuk_z2_row" path="device/zuk/z2_row" remote="github"  revision="8x-caf" />
-<project name="zuk-italian-devs/android_device_zuk_msm8996-common" path="device/zuk/msm8996-common" remote="github"  revision="8x-caf" />
-<project name="zuk-italian-devs/android_vendor_zuk" path="vendor/zuk" remote="github"  revision="lineage-15.1" />
-<project name="dd3boh/android_kernel_zuk_msm8996" path="kernel/zuk/msm8996" remote="github" revision="lineage-15.1" />
-
-<project name="AOSP-JF-MM/packages_apps_SnapdragonCamera" path="packages/apps/SnapdragonCamera" remote="github"  revision="aosp-8.1" />
```

Comment:
I don't know if remove this is ok. But remove it didn't mess my installed Camera(`org.lineageos.snap`)